salesforce-analytics-cloud/3.2/modules/ROOT/pages/salesforce-analytics-connector-reference.adoc

```diff
@@ -0,0 +1,552 @@
+* SALESFORCE-ANALYTICS:INVALID_SESSION
+* SALESFORCE-ANALYTICS:RETRY_EXHAUSTED
+* SALESFORCE-ANALYTICS:TRANSACTION
+* SALESFORCE-ANALYTICS:UNKNOWN
+
+
+[[uploadExternalDataIntoNewDataSetAndStartProcessing]]
+== Upload External Data Into New Dataset And Start Processing
+
+`<salesforce-analytics:upload-external-data-into-new-data-set-and-start-processing>`
+
+Creates a dataset, uploads data into it, and tells the Salesforce Analytics Cloud system to start processing the uploaded data.
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | |x
+| Type a| String |  Type of the records to be inserted. This is automatically computed based on metadata extracted from the provided file that contains the configuration element. |  |x
+| Records a| Array of Object |  List of records to insert. |  `#[payload]` |
+| Operation a| Enumeration, one of:
+
+** APPEND
+** DELETE
+** OVERWRITE
+** UPSERT
+|  |  |x
+| Description a| String |  |  |x
+| Label a| String |  |  |x
+| Data Set Name a| String |  |  |x
+| Edgemart Container a| String |  |  |
+| Notification Sent a| Enumeration, one of:
+
+** ALWAYS
+** FAILURES
+** NEVER
+** WARNINGS
+|  |  |
+| Notification Email a| String |  |  |
+| Target Variable a| String |  The variable name where the output of the operation is stored. |  |
+| Target Value a| String |  An expression to evaluate against the operation's output. The outcome of that expression is stored in the target variable. |  `#[payload]` |
+| Reconnection Strategy a| * reconnect
+* reconnect-forever |  A retry strategy in case of connectivity errors. |  |
+|===
+
+=== Output
+
+[%header%autowidth.spread]
+|===
+| Type a| String
+|===
+
+=== For Configurations
+
+* salesforce-analytics
+
+=== Throws
+
+* SALESFORCE-ANALYTICS:CONNECTIVITY
+* SALESFORCE-ANALYTICS:CONNECTIVITY
+* SALESFORCE-ANALYTICS:INVALID_SESSION
+* SALESFORCE-ANALYTICS:RETRY_EXHAUSTED
+* SALESFORCE-ANALYTICS:TRANSACTION
+* SALESFORCE-ANALYTICS:UNKNOWN
+
+
+== Types
+[[Tls]]
+=== Tls
+
+[cols=".^20%,.^25%,.^30%,.^15%,.^10%", options="header"]
+|======================
+| Field | Type | Description | Default Value | Required
+| Enabled Protocols a| String | A comma-separated list of protocols enabled for this context. |  |
+| Enabled Cipher Suites a| String | A comma-separated list of cipher suites enabled for this context. |  |
+| Trust Store a| <<TrustStore>> |  |  |
+| Key Store a| <<KeyStore>> |  |  |
+| Revocation Check a| * <<standard-revocation-check>>
+* <<custom-ocsp-responder>>
+* <<crl-file>> |  |  |
+|======================
+
+[[TrustStore]]
+=== Trust Store
+
+[cols=".^20%,.^25%,.^30%,.^15%,.^10%", options="header"]
+|======================
+| Field | Type | Description | Default Value | Required
+| Path a| String | The location (which will be resolved relative to the current classpath and file system, if possible) of the trust store. |  |
```

Comment:
```suggestion
| Path a| String | The location of the trust store, that if possible it will resolve relative to the current classpath and file system.. |  |
```